yt_dlp/extractor/common.py

```diff
@@ -911,13 +920,19 @@ def _download_webpage_handle(self, url_or_request, video_id, note=None, errnote=
                   returning True if it should be accepted
             Note that this argument does not affect success status codes (2xx)
             which are always accepted.
+        impersonate -- the impersonate target: can be either a string in the format of
+            CLIENT[:OS] or True (bool) if any target is sufficient
+        force_impersonate -- flag to toggle whether the request should raise an error
+            if impersonation is not possible (bool, default: False)
```

Comment:
Do you have a example use case for `force_impersonate=False`? Wouldn't this be better pattern?
```py
try:
    page = self._request(..., impersonate=True)
except ...:
    # Do stuff for bypass
    page = self._request(...)
```
as opposed to:
```py
page = self._request(..., force_impersonate=False)
if 'captcha' in page:
    # Do stuff for bypass
    page = self._request(...)
```

imo, we don't need `force_impersonate=False` unless sufficient use-cases emerge

Reply:
nvm, I misread the description